snapshot/overlay/overlay.go

```diff
@@ -114,7 +114,18 @@ func (o *Snapshotter) Prepare(key, parent string) ([]containerd.Mount, error) {
 }
 
 func (o *Snapshotter) View(key, parent string) ([]containerd.Mount, error) {
-	panic("not implemented")
+	if parent == "" {
```

Comment:
The definition of the empty string is the empty directory. I see no reason to create this special case and arbitrary restriction.